docs/commands/json.merge.md

```diff
@@ -1 +1,115 @@
-Sets or updates the JSON value at a path.
+Merge one or more `value` in a JSON document adding, updating, deleting and/or replacing a array at `path` in `key`
+
+Compliance with Json Merge Patch [RFC7396](https://datatracker.ietf.org/doc/html/rfc7396)
+
+[Examples](#examples)
+
+## Required arguments
+
+<details open><summary><code>key</code></summary>
+
+is key to merge.
+</details>
+
+<details open><summary><code>path</code></summary>
+
+is JSONPath to specify. Default is root `$`. For new Redis keys the `path` must be the root. For existing keys, when the entire `path` exists, the value that it contains is replaced with the `json` value. For existing keys, when the `path` exists, except for the last element, a new child is added with the `json` value.
+
```

Comment:
@adrianoamaral Sorry, I see that you were pretty much aligned with corresponding phrasing in `JSON.SET`, such as 
> ...For new Redis keys...

So you can revert some of my suggestions so it would be syched as much a possible with `JSON.SET`?